### DISCUSSION TITLE
How to log out

### DISCUSSION THREAD

**Discussion by SkepticMystic** [Dan6erbond/sk-auth#105] [Q&A]
Hey, thanks for making this great package :)

I am wondering how to log out a user?
Is it as simple as setting the `$session` to `null`? Or is there more that needs to be done?

**Comment by ryvdx** [comment]
I second this question... We are using Keycloak as our OAuth provider, and have a route for "end_session_endpoint" that I can't find a corresponding attribute in "OAuth2ProviderConfig" to pass this URL too?

**Reply by SkepticMystic** [reply]
Quite alot about Lucia has changed since I first asked this question.
These days, I think you would have to manually call the URL in your /signout endpoint. 
Check the latest docs to see how to set that up
